docs/demo.rst

```diff
@@ -217,6 +217,157 @@ using:
+command-line client to create a registry similar to that used by
+1kgenomes.ga4gh.org. Your system should have samtools installed, and at
+least 30GB to host the VCF and reference sets.
+
+Repo administrator CLI
+----------------------
+
+The CLI has methods for adding and removing Feature Sets, Read Group
+Sets, Variant Sets, etc. Before we can begin adding files we must first
+initialize an empty registry database. The directory that this database 
+is in should be readable and writable by the current user, as well as the 
+user running the server.
+
+.. code-block:: bash
+
+    ga4gh_repo init registry.db
+
+This command will create a file ``registry.db`` in the current working
+directory. This file should stay relatively small (a few MB for
+thousands of files).
+
+Now we will add a dataset to the registry, which is a logical container
+for the genomics data we will later add. You can optionally provide a
+description using the ``--description`` flag.
+
+.. code-block:: bash
+
+    ga4gh_repo add-dataset registry.db 1kgenomes --description "Variants from the 1000 Genomes project and GENCODE genes annotations"
+
+Add a Reference Set
+-------------------
+
+It is possible for a server to host multiple reference assemblies. Here
+we will go through all the steps of downloading and adding the FASTA
+used for the 1000 Genomes VCF.
+
+.. code-block:: bash
+
+    wget ftp://ftp.1000genomes.ebi.ac.uk//vol1/ftp/technical/reference/phase2_reference_assembly_sequence/hs37d5.fa.gz
+
+This file is provided in ``.gz`` format, which we will decompress, and
+then with samtools installed on the system, recompress it using
+``bgzip``.
+
+.. code-block:: bash
+
+    gunzip hs37d5.fa.gz
+    bgzip hs37d5.fa
+
+This may take a few minutes depending on your system as this file is
+around 3GB. Next, we will add the reference set.
+
+.. code-block:: bash
+
+    ga4gh_repo add-referenceset registry.db /full/path/to/hs37d5.fa.gz \
+      -d “NCBI37 assembly of the human genome” --ncbiTaxonId 9606 --name NCBI37 \
+      --sourceUri "ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/technical/reference/phase2_reference_assembly_sequence/hs37d5.fa.gz"
+
+A number of optional command line flags have been added. We will be
+referring to the name of this reference set ``NCBI37`` when we later add
+the variant set.
+
+Add an ontology TODO
+--------------------
+
+Ontologies provide a source for parsing variant annotations, as well as
+organizing feature types into ontology terms. This is a custom format
+created for this server.
+
+.. code-block:: bash
+
+    ga4gh_repo add-ontology registry.db /full/path/to/sequence_ontology.txt
+
+Add sequence annotations
+------------------------
+
+The GENCODE Genes dataset provides annotations for features on the
+reference assembly. The server uses a custom storage format for sequence 
+annotations, you can download a prepared set 
+`here <https://ga4ghstore.blob.core.windows.net/testing/gencode_v24lift37.db>`__.
+
+.. code-block:: bash
```

Comment:
We should delete this block now --- it'll just confuse people trying to follow the commands.